# 寸法と歩行軌道のパラメータはm単位でそろえる
walk_pattern:
  forward: 
    points: [[-0.0165,-0.065,0.0165,-0.065,1],[0,-0.050,0,-0.065,1],[0.0165,-0.065,-0.0165,-0.065,1]]
    std_vel: 0.0083
  left_turn: 
    points: [[0,-0.065,0.025,-0.065,1],[0,-0.065,0.008,-0.065,1],[0,-0.065,0.0165,-0.050,1]]
    std_omega: 0.5
leg_sizes: [0.035,0.035]
moror_ids: [4,5,6,7,3]
motor_params:
  - [1,2.6]
  - [1,2.6]
  - [1,2.6]
  - [1,2.6]
  - [0.3,0]
